cmd/nfd-master/main.go

```diff
@@ -154,5 +156,8 @@ func initFlags(flagset *flag.FlagSet) (*master.Args, *master.ConfigOverrideArgs)
 	flagset.Var(overrides.ResyncPeriod, "resync-period",
 		"Specify the NFD API controller resync period."+
 			"It has an effect when the NodeFeature API has been enabled (with -enable-nodefeature-api).")
+	overrides.MaxNumberOfConcurrentLabelers = flagset.Int("max-number-of-concurrent-labelers", 200, "Defines the maximum number of goroutines responsible of labeling nodes. "+
```

Comment:
This should be `flagset.Uint` instead of `Int` cause negative values are incorrect.

Reply:
Yes 100% true, but if the user inputs for example `-5` it would result in having `18446744073709551611 ` for the `MaxNumberOfConcurrentLabelers`. I would suggest working with `math.Abs` to handle negative values. wdyt?